chatroom_handoff.html

```diff
@@ -0,0 +1,20 @@
+<head>
+  <link rel="stylesheet" href="http://127.0.0.1:8080/dist/Chatroom.css" />
+</head>
+<body>
+  <div class="chat-container"></div>
+
+  <script src="http://127.0.0.1:8080/dist/Chatroom.js"/></script>
+  <script type="text/javascript">
+    var chatroom = new window.Chatroom({
+      host: "http://192.168.64.2/webhooks/rest/webhook",
```

Comment:
Should this point to localhost and is `handoffhost` still used?

Reply:
ooh no it should not be like this! I'll fix it

Reply:
Hmm, also realize we need more straightforward instructions for actually starting this